### PR TITLE
TplNames renamed TplName ,fix #1229,Remember modify bee tool.

### LINKED ISSUE #1229: Misspelling of TplNames?
Cant help to notice what seems to me to be a strange naming of "this.TplNames" (plural "-s").  Why?  Isn't this always one file or template?

### PR DESCRIPTION
**Pull request by ysqi**
Are you agree rename "tplNames" ?  Need update bee if merged  !

### CODE REVIEW

File: namespace_test.go [hunk 1]
```diff
@@ -61,8 +61,8 @@ func TestNamespaceNest(t *testing.T) {
 	ns.Namespace(
 		NewNamespace("/admin").
 			Get("/order", func(ctx *context.Context) {
-				ctx.Output.Body([]byte("order"))
-			}),
+			ctx.Output.Body([]byte("order"))
+		}),
```
Comment (astaxie) [inline namespace_test.go]: why this fmt changed?

**Comment by ysqi** [comment]
run go fmt

File: namespace_test.go (hunk 1 above)
Comment (ysqi) [inline namespace_test.go]: need remove namespace_test.go?

**Comment by rhcarvalho** [comment]
Hi! This and other recent PRs are changing the public API of beego main types.
Even ["Hello World" examples](https://github.com/beego/samples/blob/master/WebIM/controllers/websocket.go#L41) that depend on beego controllers got broken with this change :cry:

**Pull request merged**